torch/csrc/utils/python_dispatch.cpp

```diff
@@ -202,6 +202,15 @@ void initDispatchBindings(PyObject* module) {
     c10::Dispatcher::singleton().checkInvariants();
   });
 
+  m.def("_dispatch_has_kernel_for_dispatch_key", [](const char* name, const char* dispatch) -> bool {
+    auto op = c10::Dispatcher::singleton().findOp(torch::jit::parseName(name));
+    if (!op) {
+      return false;  // TODO: or raise an error?
```

Comment:
So it turns out this is very annoying, because TorchScript adds a bunch of extra overloads to operators that aren't actually in the dispatcher.  I ended up with this in the end:

```
                if (
                    not disable_meta
                    # TorchScript dumps a bunch of extra nonsense overloads
                    # which don't have corresponding dispatcher entries, we need
                    # to filter those out
                    and torch._C._dispatch_has_kernel(name)
                    and not torch._C._dispatch_has_kernel_for_dispatch_key(name, 'Meta')
                ):  
                    meta_lib.impl(op_overload, f)
```

Reply:
Yikes, thanks for the explanation